joblib/parallel.py

```diff
@@ -152,10 +152,20 @@ class parallel_backend(object):
     multiprocessing and loky may not be available, in which case joblib
     defaults to threading.
 
-    In addition, if the `dask` and `distributed` Python packages are installed,
-    it is possible to use the 'dask' backend for better scheduling of nested
-    parallel calls without over-subscription and potentially distribute
-    parallel calls over a networked cluster of several hosts.
+    You can also use the `Dask <https://docs.dask.org/en/stable/>`_ joblib
+    backend to distribute work across machines. This works well with
+    scikit-learn estimators with the ``n_jobs`` parameter, for example::
```

Comment:
I am wondering whether this would make more sense to have an example only using joblib like the ray one below ...

Reply:
The benefice here is to show that this changes even in downstream libraries that use `joblib`.
But maybe we could link to the example from the example gallery?

Reply:
Yep, but not sure that the `parallel_backend` docstring is the best place to do this? Having said that I am completely fine leaving it like this for this PR.

Looking a bit more, it seems like maybe there should be a backend paragraph after https://joblib.readthedocs.io/en/latest/parallel.html#thread-based-parallelism-vs-process-based-parallelism. This paragraph right now mentions the Spark backend at the end but does not mention the other ones ...

Reply:
I opted to add the Dask and Ray backend options to `parallel.rst` and also included a link to the example in the gallery

Reply:
Thinking again, I agree with @lesteve that this might not be the best place but let's change the example in a follow-up PR.